go_expvar/README.md

```diff
@@ -1,87 +1,75 @@
-# Go_expvar Integration
+# Go Expvar Integration
 
-## Overview
+# Overview
 
-Get metrics from go_expvar service in real time to:
+Track the memory usage of your Go services and collect metrics instrumented from Go's expvar package.
 
-* Visualize and monitor go_expvar states
-* Be notified about go_expvar failovers and events.
+If you prefer to instrument your Go code with [dogstats-go](https://github.com/DataDog/datadog-go) instead, you can still use this integration to collect memory-related metrics.
```

Comment:
Yes it does. I was supposing that users might use one or other - not both - but I think you're right: some people may choose to use both. I'll remove 'instead' 👍